lldb/source/Plugins/SymbolFile/DWARF/DebugNamesDWARFIndex.cpp

```diff
@@ -9,7 +9,7 @@
 #include "Plugins/SymbolFile/DWARF/DebugNamesDWARFIndex.h"
 #include "Plugins/SymbolFile/DWARF/DWARFDebugInfo.h"
 #include "Plugins/SymbolFile/DWARF/DWARFDeclContext.h"
-#include "Plugins/SymbolFile/DWARF/SymbolFileDWARFDwo.h"
+#include "Plugins/SymbolFile/DWARF/LogChannelDWARF.h"
```

Comment:
Is this actually needed?

Reply:
Not exactly. I wanted to remove (unused) SymbolFileDWARFDwo since it was potentially signalling breaking of abstractions, and then I got carried away include gardening (my editor now highlights unused headers :/).

Let me put those in separately.